server/conf/application.conf

```diff
@@ -678,8 +678,6 @@ feature_flag_overrides_enabled = ${?FEATURE_FLAG_OVERRIDES_ENABLED}
 # address correction
 esri_address_correction_enabled = false
 esri_address_correction_enabled = ${?ESRI_ADDRESS_CORRECTION_ENABLED}
-
-# address verification
-esri_address_service_area_validation_enabled = false
-esri_address_service_area_validation_enabled = ${?ESRI_ADDRESS_SERVICE_AREA_VALIDATION_ENABLED}
-
```

Comment:
issue (blocking): I think we still need to set these in the base conf file for prod envs

Reply:
re-added config values